doc/source/docs/user_guide/mapping.rst

```diff
@@ -154,6 +154,17 @@ However, passing ``missing_kwds`` one can specify the style and label of feature
         },
     );
 
+Other map customizations
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Maps usually do not have to have axis labels. You can turn them off by ``ax.set_axis_off()`` or ``ax.axis("off")``.
```

Comment:
```suggestion
Maps usually do not have to have axis labels. You can turn them off using ``set_axis_off()`` or ``axis("off")`` axis methods.
```

I think this may be better because `ax` is just an arbitrary variable.